emily/handler/src/api/handlers/deposit.rs

```diff
@@ -267,6 +270,9 @@ pub async fn update_deposits(
         context: EmilyContext,
         body: UpdateDepositsRequestBody,
     ) -> Result<impl warp::reply::Reply, Error> {
+        // Get the api state and error if the api state is claimed by a reorg.
```

Comment:
I'm going to add it in a README.md.

Since this finished the MVP features of the Emily API my next PRs are:
 - OpenAPI client (Currently putting into a PR)
 - Add Readme.md
 - Add node watcher
 - Deploy in cloud